src/decoder.rs

```diff
@@ -255,9 +253,9 @@ fn decode_regular(rsm: RawSourceMap) -> Result<SourceMap> {
         if !source_root.is_empty() {
             let source_root = source_root.trim_right_matches('/');
             sources = sources.into_iter().map(|x| {
-                if x.len() > 0 && (x.starts_with('/') ||
-                                   x.starts_with("http:") ||
-                                   x.starts_with("https:")) {
+                if x.is_empty() &&
```

Comment:
I'd assume it's faster to check the length and short-circuit the branch there. (Can't be sure without looking at the asm though, LLVM is pretty clever.)